src/pages/home/index.js

```diff
@@ -66,25 +67,25 @@ function Sponsors() {
     <div className={styles.slide}>
       <div id={styles.sponsorBkg}>
         <h2>Sponsors</h2>
-        <div className={styles.logoSection}>
+        <div className={styles.logoSection}> 
           {sponsorLogo(styles.rbc, 'https://www.rbcroyalbank.com', Rbc)}
         </div>
-        <div className={styles.logoSection}>
-          {sponsorLogo(styles.ciena, 'https://www.ciena.ca', Ciena)}
+        <div className={styles.logoSection}> 
+          {sponsorLogo(styles.ciena, 'https://www.ciena.ca', Ciena)} 
           {sponsorLogo(styles.cse, 'https://www.cse-cst.gc.ca/en', Cse)}
         </div>
-        <div className={styles.logoSection}>
+        <div className={styles.logoSection}> 
           {sponsorLogo(styles.fullscript, 'https://fullscript.com', Fullscript)}
           {sponsorLogo(styles.lighthouseLabs, 'https://www.lighthouselabs.ca', LighthouseLabs)}
         </div>
-        <div className={styles.logoSection}>
+        <div className={styles.logoSection}> 
           {sponsorLogo(styles.marchNetworks, 'https://marchnetworks.com', MarchNetworks)}
         </div>
-        <div className={styles.logoSection}>
+        <div className={styles.logoSection}> 
           {sponsorLogo(styles.balsamiq, 'https://balsamiq.com', Balsamiq)}
           {sponsorLogo(styles.fdmGroup, 'https://fdmgroup.com', FdmGroup)}
         </div>
-        <div className={styles.logoSection}>
+        <div className={styles.logoSection}>  
```

Comment:
I'm not sure how you managed to add white space here, but please remove it (for my sake, it's annoying).

Reply:
Not sure how that got there either. Addressed in latest commit.